src/compile/selection/interval.ts

```diff
@@ -94,9 +97,18 @@ const interval:SelectionCompiler = {
 
       y2: extend({}, y !== null ?
         {scale: model.scaleName(Y), signal: `${name}[${y}].extent[1]`} :
-        {field: {group: 'height'}}),
+        {field: {group: 'height'}})
     };
 
+    if (selCmpt.resolve === 'global') {
+      keys(update).forEach(function(key) {
+        update[key] = [{
```

Comment:
Maybe add some comments here?
A few things I don't understand:
- I assume `.unit` mean something in Vega Expression that's useful for selection resolution?  It would be great if we could know what it means.
- Why `${store}`needs to refer to `[0]` ?